pkg/endpoint/bpf.go

```diff
@@ -1286,6 +1286,21 @@ func (e *Endpoint) RequireEndpointRoute() bool {
 	return e.DatapathConfiguration.InstallEndpointRoute
 }
 
+// GetPolicyVerdictLogFilter returns the PolicyVerdictLogFilter that would control
+// the creation of policy verdict logs. Value of VerdictLogFilter needs to be
+// consistent with how it is used in policy_verdict_filter_allow() in bpf/lib/policy_log.h
+func (e *Endpoint) GetPolicyVerdictLogFilter() uint {
+	// The default value must be non-zero for EBPF to compile this in
```

Comment:
This should only be necessary for the template, so if you added the same function `GetPolicyVerdictLogFilter()` for the `templateCfg` type which set a dummy non-zero value then it should be safe to clear this bit clear for the actual endpoints.

Reply:
Updated.